### PR TITLE
Remove torchvision from dependency list

### PR DESCRIPTION
**Pull request by aleksei-kashapov**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -69,10 +69,9 @@ def find_version(*file_paths):
 
 version_string = "{}{}".format(sys.version_info[0], sys.version_info[1])
 
-INSTALL_REQUIRES.extend(["torch", "torchvision"])
+INSTALL_REQUIRES.extend("torch")
 
 TORCH_VERSION = "1.7.0"
-TORCHVISION_VERSION = "0.8.1"
 CUDA_VERSION = "102"
 IS_CUDA_VER_DEFAULT_FOR_CURRENT_TORCH_VER = True
```
Comment (FyodorK) [inline setup.py:77]: Why we need TORCHVISION_SOURCE_URL_TEMPLATE in 80 line if it is not used any more?

Comment (aleksei-kashapov) [reply setup.py:78]: you are right; removed

File: setup.py [hunk 2]
```diff
@@ -69,10 +69,9 @@ def find_version(*file_paths):
 
 version_string = "{}{}".format(sys.version_info[0], sys.version_info[1])
 
-INSTALL_REQUIRES.extend(["torch", "torchvision"])
+INSTALL_REQUIRES.extend("torch")
```
[the remark this answers is not in the document]
Comment (aleksei-kashapov) [reply setup.py]: Changed to a list

File: examples/semantic_segmentation/README.md [hunk 3]
```diff
@@ -9,6 +9,14 @@ This sample demonstrates DL model compression capabilities for semantic segmenta
...
 - Tensorboard output
 
+## Installation
+
+To work with the sample you should install the corresponding Python package dependencies
+
+```
+pip install examples/requirements.txt
+```
+
```
Comment (vshampor) [inline examples/semantic_segmentation/README.md:19]: We already have this in root README.md. It doesn't hurt to give another hint at sample README level, though, but you should do this for each sample then.

**Review by vshampor — CHANGES_REQUESTED** [review]

File: examples/semantic_segmentation/README.md (hunk 3 above)
Comment (FyodorK) [reply examples/semantic_segmentation/README.md:19]: mb `pip  install **-r**  examples/requirement.txt` ?

Comment (aleksei-kashapov) [reply examples/semantic_segmentation/README.md:19]: @vshampor It seems to be more convenient for users.
 I did for classification, detection and segmentation. Do we have more?

Comment (aleksei-kashapov) [reply examples/semantic_segmentation/README.md:19]: > mb `pip install **-r** examples/requirement.txt` ?

Should be. Need to change main Readme

Comment (vshampor) [reply examples/semantic_segmentation/README.md:19]: @aleksei-kashapov please adjust the root README as well within the scope of this PR.

Comment (aleksei-kashapov) [reply examples/semantic_segmentation/README.md:19]: Done

**Review by vshampor — APPROVED** [review]

**Pull request merged**